src/layouts/DashboardLayout/Navbar/Navbar.tsx

```diff
@@ -26,7 +27,7 @@ const Navbar = () => {
           <Typography variant="h2" fontSize={{ md: '1.5rem', xs: '1rem' }}>
             {t('dashboard.welcome_back', {
               // TODO - get from auth slice
```

Comment:
we can remove this todo ;) :*

Reply:
you're right, will remove it in the next PR 😉